forge-gui/res/cardsfolder/upcoming/servant_of_the_stinger.txt

```diff
@@ -0,0 +1,9 @@
+Name:Servant of the Stinger
+ManaCost:1 B
+Types:Creature Human Warlock
+PT:1/3
+K:Deathtouch
+T:Mode$ DamageDone | ValidSource$ Card.Self | Execute$ TrigSearch | CombatDamage$ True | ValidTarget$ Player | CheckDefinedPlayer$ You.committedCrimeThisTurn | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, if you've committed a crime this turn, you may sacrifice Servant of the Stinger. If you do, search your library for a card, put it into your hand, then shuffle. (Targeting opponents, anything they control, and/or cards in their graveyards is a crime.)
```

Comment:
you could just do CheckSVar$ here to reuse the other syntax instead? 🤔

Reply:
Agh sorry I thought about it and decided to leave the PlayerProperty

I'd be shocked if we don't see a "each player who committed a crime this turn does X" especially in commander

Reply:
sure!